### PR TITLE
Dvc data fix

### PR DESCRIPTION
**Pull request by moudemans**
DVC bug fix for raw data

## Description
Dvc now is able to provide the raw data to the user

## Related Issue
Quick fix, no issue assigned

### CODE REVIEW

File: data/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/raw
```
Comment (Gjel) [inline data/.gitignore:1]: Why is only the raw folder in this gitignore? If this is in, must we then also put the others in there?

Comment (Gjel) [reply data/.gitignore:1]: shouldn't we do it the same way as the interim and processed folders?

**Comment by FrostMegaByte** [comment]
I performed a `dvc pull` and it seemed to work, so your fix is nice!

File: data/.gitignore (hunk 1 above)
Comment (moudemans) [reply data/.gitignore]: Fixed it by moving it to the raw dir and iognoring the datafiles, rather than the directory itself

**Pull request merged**